homework/index.js

```diff
@@ -0,0 +1,21 @@
+const repoSelectElement = document.getElementById('repositories');
+const unContainer = document.querySelector('.right__div-username');
+const reposContainer = document.querySelector('.right__div-repos');
+const avatarContainer = document.querySelector('.right__div-avatar');
+
+
+const CONTRIBUTORS_URL = async () => {
+  const API_CALL = await fetch(
+    'https://api.github.com/repos/HackYourFuture/tdd-game/contributors',
+  );
+  const data = await API_CALL.json();
+  return { data };
+};
+const showContributorsDetails = () => {
+  CONTRIBUTORS_URL(repoSelectElement.value).then((response) => {
+    unContainer.innerHTML = `username:<span class="right_div-value">${response.data.login}</span>`;
```

Comment:
reponse.data contains the array of contributors (or, if, as suggested above, you drop the wrapping object, you will receive the array of contributors directly). You'll want to iterate over that, and _for each_ contributor in the array, add a `<li>` inside your `<ul>` (as explained in the html).